.gitignore

```diff
@@ -1,4 +1,5 @@
 vendor/
 composer.lock
 tests/FacebookTestCredentials.php
+.idea/
```

Comment:
You should remove this and use [global ignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore)